config.md

```diff
@@ -139,6 +139,40 @@ For POSIX platforms the `mounts` structure has the following fields:
 ]
 ```
 
+### <a name="configLinuxMounts" />Linux Mounts
+
+For Linux, the `mounts` structure mounts MAY have the following additional fields:
+
+* **`attr`** (object, OPTIONAL) specifies `mount_setattr(2)` attributes. Requires kernel 5.12 or later.
+    * **`flags`** (array of strings, OPTIONAL) specifies the `flags` argument of `mount_setattr(2)` syscall.
+       e.g.,  "AT_RECURSIVE"
+    * **`attr_set`** (array of strings, OPTIONAL) specifies the `attr_set` field of `mount_attr` struct.
+       e.g., "MOUNT_ATTR_RDONLY"
+    * **`attr_clr`** (array of strings, OPTIONAL) specifies the `attr_clr` field of `mount_attr` struct.
+    * **`propagation`** (string, OPTIONAL) specifies the `propagation` field of `mount_attr` struct.
+       See [Rootfs Mount Propagation](./config-linux.md#configLinuxRootfsMountPropagation) for the string format.
+
+<!-- TODO: consider supporting mount_attr.userns_fd -->
+
+### Example
+
+```json
+"mounts": [
+    {
+        "destination": "/ro",
+        "type": "none",
+        "source": "/volumes/ro",
+        "options": ["rbind"],
+        "attr": {
+          "flags": ["AT_RECURSIVE"],
+          "attr_set": ["MOUNT_ATTR_RDONLY"],
+          "attr_clr": ["MOUNT_ATTR_NOEXEC"],
+          "propagation": "private"
+        }
```

Comment:
I'm a bit concerned about how this is just a 1:1 mapping of `mount_setattr` (although I realise we already crossed that path on many other options). The advantage is that it's flexible, but it we only need it for the `recursively-readonly`, I'm wondering if it would be good to design around that case, making it somewhat more less verbose?

It makes it a bit confusing in some areas, e.g., `options` would take  `(r)private`, `ro` and `noexec`, but `attr` takes `MOUNT_ATTR_RDONLY` and `MOUNT_ATTR_NOEXEC` (in addition to using a "diff" of attributes to _add_ and to _remove_).

Reply:
> The advantage is that it's flexible, but it we only need it for the recursively-readonly, I'm wondering if it would be good to design around that case, making it somewhat more less verbose?

I don't think this is going to be recursively-readonly, though we can prioritize recursively-readonly over other potential usecases

> It makes it a bit confusing in some areas, e.g., options would take (r)private, ro and noexec, but attr takes MOUNT_ATTR_RDONLY and MOUNT_ATTR_NOEXEC (in addition to using a "diff" of attributes to add and to remove).

`MOUNT_ATTR_XXX` form corresponds to `RLIMIT_XXX` form that has already existed.